src/common.cpp

```diff
@@ -124,8 +291,141 @@ namespace dynamicgraph
 	  q_urdf[idx]=q_sot[m_urdf_to_sot[idx]];	
 	return true;
       }
+      
+      bool RobotUtil::
+      velocity_urdf_to_sot(Eigen::ConstRefVector v_urdf, Eigen::RefVector v_sot)
+      {
+	assert(q_urdf.size()==m_nbJoints+6);
+	assert(q_sot.size()==m_nbJoints+6);
+	
+	if (m_nbJoints==0)
+	  {
+	    SEND_MSG("velocity_urdf_to_sot should be called", MSG_TYPE_ERROR);
+	    return false;
+	  }
+
+        v_sot.head<6>() = v_urdf.head<6>();
+        joints_urdf_to_sot(v_urdf.tail(m_nbJoints), 
+			   v_sot.tail(m_nbJoints));
+	return true;
+      }
+
+      bool RobotUtil::
+      velocity_sot_to_urdf(Eigen::ConstRefVector v_sot, Eigen::RefVector v_urdf)
+      {
+	assert(q_urdf.size()==m_nbJoints+6);
```

Comment:
same thing fixed in https://github.com/stack-of-tasks/sot-torque-control/pull/22